src/PlaneWaveBasis.jl

```diff
@@ -81,6 +86,9 @@ struct PlaneWaveBasis{T <: Real}
     # Independent of the `use_symmetry` option
     symmetries::Vector{SymOp}
 end
+# prevent broadcast on pwbasis
+import Base.Broadcast.broadcastable
```

Comment:
I see why you do this, but I tend to prefer imports at the top of the file.

Reply:
That doesn't really make sense here: better to have it locally where it's used